src/Bicep.Core/Registry/AzureContainerRegistryManager.cs

```diff
@@ -182,12 +182,18 @@ private static async Task<Stream> ProcessManifest(ContainerRegistryContentClient
             }
 
             ProcessConfig(manifest.Config);
-            if (manifest.Layers.Length != 1)
+
+            // First layer must be our bicep artifact layer.  Ignore any other layers to allow for future expansion
+            //   (but note that v0.18 and earlier  throw if they find multiple layers, so it will be a breaking change
+            //   with those Bicep versions if we add additional layers).  In the meantime, additional data needs to be attached
+            //   in a new artifact.
+            if (manifest.Layers.Length < 1)
             {
-                throw new InvalidModuleException("Expected a single layer in the OCI artifact.");
+                throw new InvalidModuleException("Expected at least one layer in the OCI artifact.");
             }
 
-            var layer = manifest.Layers.Single();
+            // Ignore other layers for now
+            var layer = manifest.Layers.First(); 
```

Comment:
>First [](http://example.com/codeflow?start=40&length=5)

We shouldn't depend on the order of layers. I think we should throw if we don't find a single layer with the expected media type. If we find exactly one at index 5, that seems fine. We should also throw if we find multiple layers with the expected media type since the current version doesn't know how to handle that.